screens/FarmOrchardScreen.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useState } from "react";
+import MapView, { PROVIDER_GOOGLE, Polyline } from "react-native-maps";
+import {
+  View,
+  StyleSheet,
+  Dimensions,
+  TouchableOpacity,
+  Text,
+} from "react-native";
+import tailwind from "tailwind-react-native-classnames";
+import {
+  useNavigation,
+  useIsFocused,
+  useRoute,
+} from "@react-navigation/native";
+
+import axios from "axios";
+
+export default function FarmOrchardScreen() {
+  const GOOGLE_MAPS_APIKEY = "AIzaSyBBkDvVVuQBVSMOt8wQoc_7E-2bvDh2-nw";
+
+  const isFocused = useIsFocused();
+
+  const navigation = useNavigation();
+
+  const route = useRoute();
+
+  const { id, name } = route?.params;
+
+  const [orchardData, setOrchardData] = useState([]);
+
+  const access_token = "1566394169B0EJX2MGAVKVUGGKEMKZBMND9A7VCR";
+
+  const [coordinatePoints, setCoordinatePoints] = useState([]);
+
+  const [startingPoint] = coordinatePoints;
+
+
+  const startCoordinates = {
+    latitude: startingPoint?.latitude,
+    longitude: startingPoint?.longitude,
+    latitudeDelta: 0.005,
+    longitudeDelta: 0.005,
+  };
+
+  const getOrchards = async () => {
+    const res = await axios
+      .get(
+        `https://sherlock.aerobotics.com/developers/orchards/?farm_id=${id}`,
+        {
+          headers: {
+            Authorization: `${access_token}`,
+          },
```

Comment:
If you read the axios docs, you'll see there is a way to configure axios so you don't have to keep passing this header in.